gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -164,13 +156,13 @@
               </div>
             </div>
           </div>
-          <div class="form-group" ng-if="willBeReintroducedBy">
-            <label translate for="removal">Device Removal</label>
-              <p class="text-danger">When you remove this device, it might be automatically reintroduced by {{ willBeReintroducedBy }}.</p>
-          </div>
         </div>
       </div>
     </form>
+    <div class="form-group" ng-if="willBeReintroducedBy">
+      <label translate for="removal">Device Removal</label>
+      <p class="text-danger">When you remove this device, it might be automatically reintroduced by {{ willBeReintroducedBy }}.</p>
```

Comment:
```suggestion
      <p class="text-danger" translate translate-value-introducer="{{ willBeReintroducedBy }}">When you remove this device, it might be automatically reintroduced by {%introducer%}.</p>
```

Reply:
The interpolation on introducer does not seem to be working, are we missing something?